config/database.php

```diff
@@ -162,11 +160,11 @@
 	*/
 
 	'redis' => [
-		'client' => env('REDIS_CLIENT', 'phpredis'),
+		'client' => 'phpredis',
 
 		'options' => [
 			'cluster' => env('REDIS_CLUSTER', 'redis'),
-			'prefix' => env('REDIS_PREFIX', Str::slug(env('APP_NAME', 'laravel'), '_') . '_database_'),
+			'prefix' => env('REDIS_PREFIX', Str::slug(env('APP_NAME', 'Lychee'), '_') . '_database_'),
```

Comment:
I don't know if the capitalisation is significant here, but it's worth considering.